setup.py

```diff
@@ -86,7 +86,8 @@
     "filelock",
     "flax>=0.4.1",
     "hf-doc-builder>=0.3.0",
-    "huggingface-hub>=0.10.0",
+    "huggingface-hub>=0.13.0",
+    "requests-mock==1.10.0",
```

Comment:
This testing library is super useful to check how many HEAD, GET requests were made, is popular and very lightweight, so think we can add it here to help with testing.